src/sass/pages/_cookies.scss

```diff
@@ -0,0 +1,7 @@
+.cookies-page-dfe-sign-in {
```

Comment:
There might be a way to get this done in layout.scss and it will be available to any other component.
`@for $i from 1 through 12 {
  .col-#{$i} {
    @include grid-column($i / 12);
  }
}`

Make it something like:
`@for $i from 1 through 12 {
  .col-#{$i} {
    @include grid-column($i / 12);
    .no-padding {
      padding: 0;
    }
  }
}`

And use it like class="col-8 no-padding"
Then we don't need that cookies-page-dfe-sign-in class at all (I think)

Reply:
Yeah; see I thought about this at the time; though I was trying to be mindful of not breaking anyone else's sites with global changes; because I know this repo shared outside of DSI, too.
What are your thoughts 😃

Reply:
The change I propose should not break existing code but in any case if you want to stay on the safe side keep it as is 
👍